ramlfications/loader.py

```diff
@@ -35,7 +40,78 @@ def _yaml_include(self, loader, node):
                 return inputfile.read()
 
         with open(file_name) as inputfile:
-            return yaml.load(inputfile, self._ordered_loader)
+            parsed = yaml.load(inputfile, self._ordered_loader)
+            if file_ext == ".json":
+                parsed = self._parse_json_refs(
+                    parsed, os.path.dirname(file_name)
+                )
+            return parsed
+
+    def _lookup_json_ref(self, ref_key, base_path=None):
+        ref_source, ref_fragment = ref_key.split("#/")
+
+        # Determine the correct identifier
+        relative_path = os.path.join(base_path, ref_source)
+        file_name, uri = None, None
+        if ref_source.startswith('/'):  # Absolute path
+            file_name = ref_source if os.path.isfile(ref_source) else None
+        elif bool(urlparse(ref_source).scheme):  # URL
+            uri = ref_source
+        elif base_path and os.path.exists(relative_path):
+            file_name = relative_path
+        else:
+            raise LoadRAMLError(
+                "Invalid JSON ref: unable to process {src}".format(
+                    src=ref_source
+                )
+            )
+
+        # Load the ref if we dont have it
+        if ref_source not in self.refs.keys():
```

Comment:
Do `file:///` URLs have to be full (a opposed to relative) paths? If so, that might be suboptimal. Unless this also builds that full path.